src/modules/villages/village_repository.ts

```diff
@@ -0,0 +1,15 @@
+import database from '../../config/database';
+import { Village as Entity } from './village_entity';
+
+export namespace Village {
+  export const Villages = () => database<Entity.Struct>('villages')
+
+  export const findAllWithLocation = () => {
+    const query = Villages()
+      .select('id', 'name', 'level', 'location')
+      .where('is_active', true)
+      .orderBy('name', 'asc')
+
+    return query
+  }
```

Comment:
Seems like we also need to return the category name
![image](https://user-images.githubusercontent.com/52171676/136889592-84d4317b-6d07-4193-b707-39f0ab80c1fe.png)